Jenkinsfile

```diff
@@ -20,6 +20,11 @@ def runDevelopmentTests(){
   sh 'make developmentTests'
 }
 
+def runDevelopmentChromaticTests(){
+  sh 'make install'
```

Comment:
@mbzain76 do we need to run the `make install` step in these 3 methods? running this step prior just once will save us some time.

Reply:
we could move the `make install` command to just above these two parallel steps.

Reply:
`make install` should be added to every stage, it was failing without it.